### PR TITLE
Made it not show the path.

### PR DESCRIPTION
**Pull request by weeryan17**
Yegh showing the path didn't look good.

### CODE REVIEW

File: src/main/java/com/bwfcwalshy/flarebot/github/GithubListener.java [hunk 1]
```diff
@@ -26,8 +26,9 @@ public void handle(PushEvent e) {
                 StringBuilder sb = new StringBuilder();
                 sb.append("```md\n");
                 int i = 1;
-                for (String file : commit.getAdded()) {
+                for (String filePath : commit.getAdded()) {
                     if (!(i++ >= 5)) {
```
Comment (WalshyDev) [inline src/main/java/com/bwfcwalshy/flarebot/github/GithubListener.java]: Change this to greater than also you can just do if(i <= 5) why complicate things?

**Comment by weeryan17** [comment]
Arsen apparently did it already.

File: src/main/java/com/bwfcwalshy/flarebot/github/GithubListener.java [hunk 2]
```diff
@@ -44,8 +45,9 @@ public void handle(PushEvent e) {
                 StringBuilder sb = new StringBuilder();
                 sb.append("```md\n");
                 int i = 1;
-                for (String file : commit.getRemoved()) {
+                for (String filePath : commit.getRemoved()) {
                     if (!(i++ >= 5)) {
```
Comment (WalshyDev) [inline src/main/java/com/bwfcwalshy/flarebot/github/GithubListener.java]: Same here

File: src/main/java/com/bwfcwalshy/flarebot/github/GithubListener.java [hunk 3]
```diff
@@ -62,8 +64,9 @@ public void handle(PushEvent e) {
                 StringBuilder sb = new StringBuilder();
                 sb.append("```md\n");
                 int i = 1;
-                for (String file : commit.getModified()) {
+                for (String filePath : commit.getModified()) {
                     if (!(i++ >= 5)) {
```
Comment (WalshyDev) [inline src/main/java/com/bwfcwalshy/flarebot/github/GithubListener.java]: Same here

**Pull request merged**